Automattic/wp-calypso
# Theme Upload: Improve the notifications for AT transfer with theme upload

This PR improves the AT notifications and transfer errors for the theme upload interface. 

Previously, the user was not presented with any input (or was behaving incorrectly) that signals:
- An error has occurred while setting up the theme
- The theme upload & AT transfer was finished

Additionally, I also removed the WP-Admin redirect on Atomic because it creates an on-optimal user experience. If the transfer is completed (or failed) and the user refreshes the page, they would be redirected to a totally different UI. 



Related to https://github.com/Automattic/wp-calypso/issues/93581 and https://github.com/Automattic/wp-calypso/issues/93582

## Proposed Changes

* Display correctly the in-progress notice
* Display an error on theme configuration
* Remove the WP-Admin redirect on AT

## Why are these changes being made?


* bug fix

## Testing Instructions



* Apply D160747-code on your sandbox
* Use the live link and go to /themes/upload/your-business-plan-site
* Upload a zip file with a theme
* You should get the in-progress notice
* The theme UI card should be displayed after the upload 

## Pre-merge Checklist
